installation/language/de-DE/de-DE.ini

```diff
@@ -3,123 +3,92 @@
+INSTL_DATABASE="Datenbank-Konfiguration"
 INSTL_DATABASE_ERROR_POSTGRESQL_QUERY="PostgreSQL-Datenbankabfrage fehlgeschlagen."
 INSTL_DATABASE_HOST_DESC="Üblicherweise ist dies „localhost“ oder ein vorgegebener Name des Webhosters."
 INSTL_DATABASE_HOST_LABEL="Servername"
-INSTL_DATABASE_HOST_IS_NOT_LOCALHOST_GENERAL_MESSAGE="Es soll ein anderer Datenbankserver als „localhost“ verwendet werden. Aus Sicherheitsgründen muss die Eigentümerschaft des Webhostings überprüft werden. Weitere Informationen dazu können <a href=\"%s\">in der Dokumentation</a> gefunden werden."
-INSTL_DATABASE_HOST_IS_NOT_LOCALHOST_DELETE_FILE="Um die Eigentümerschaft zu bestätigen muss die Datei „%s“, die soeben im „installation“-Verzeichnis von Joomla! erstellt wurde, wieder gelöscht werden."
 INSTL_DATABASE_HOST_IS_NOT_LOCALHOST_CREATE_FILE="Aufgrund eines unbekannten Fehlers kann die Datei nicht automatisch erstellt werden. Daher ist die Datei mit dem Namen „%s“ manuell zu erstellen und in das Verzeichnis „installation“ hochzuladen."
-INSTL_DATABASE_NAME_DESC="Einige Webhoster erlauben nur eine Datenbank pro Website. In diesem Fall sollte ein eindeutiger Tabellenpräfix für Joomla! gewählt werden."
-INSTL_DATABASE_NAME_LABEL="Datenbankname"
-INSTL_DATABASE_NO_SCHEMA="Es existiert kein Datenbankschema für diesen Datenbanktyp!"
-INSTL_DATABASE_OLD_PROCESS_DESC="„Sichern“ oder „Löschen“ bereits vorhandener Joomla!-Tabellen mit demselben Tabellenpräfix."
-INSTL_DATABASE_OLD_PROCESS_LABEL="Alte Datenbanktabellen"
-INSTL_DATABASE_PASSWORD_DESC="Für die Sicherheit der Website sollte immer ein Datenbankpasswort gesetzt sein!"
-INSTL_DATABASE_PASSWORD_LABEL="Passwort"
-INSTL_DATABASE_PREFIX_DESC="Einen Tabellenpräfix vergeben oder den <strong>zufällig generierten</strong> belassen. Idealerweise besteht dieser aus 3 bis 4 Zeichen, enthält nur alphanumerische Zeichen und MUSS mit einem Unterstrich enden. <strong>Es MUSS sichergestellt sein, das der Präfix nicht schon von anderen Tabellen genutzt wird!</strong>"
-INSTL_DATABASE_PREFIX_LABEL="Tabellenpräfix"
-INSTL_DATABASE_PREFIX_MSG="Der Tabellenpräfix muss mit einem Buchstaben beginnen, danach können optional alphanumerische Zeichen und ein Unterstrich folgen."
-INSTL_DATABASE_TYPE_DESC="Dies ist normalerweise „MySQLi“"
-INSTL_DATABASE_TYPE_LABEL="Datenbanktyp"
-INSTL_DATABASE_USER_DESC="Entweder ein selbst eingerichteter Benutzername oder ein vorgegebener Benutzername des Webhosters."
-INSTL_DATABASE_USER_LABEL="Benutzername"
+INSTL_DATABASE_HOST_IS_NOT_LOCALHOST_DELETE_FILE="Um die Eigentümerschaft zu bestätigen muss die Datei „%s“, die soeben im „installation“-Verzeichnis von Joomla! erstellt wurde, wieder gelöscht werden."
+INSTL_DATABASE_HOST_IS_NOT_LOCALHOST_GENERAL_MESSAGE="Es soll ein anderer Datenbankserver als „localhost“ verwendet werden. Aus Sicherheitsgründen muss die Eigentümerschaft des Webhostings überprüft werden. Weitere Informationen dazu können <a href=\'%s\'>in der Dokumentation</a> gefunden werden."
```

Comment:
INSTL_DATABASE_HOST_IS_NOT_LOCALHOST_GENERAL_MESSAGE="Es soll ein anderer Datenbankserver als „localhost“ verwendet werden. Aus Sicherheitsgründen muss dazu die Eigentümerschaft des Webhostings überprüft werden. Weitere Informationen sind in der <a href=\'%s\'>Dokumentation</a> zu finden."

Reply:
Angleichung an bestehende Verweise auf die Dokumentation.